### PR TITLE
Add New Environment via app settings

### PR DESCRIPTION
**Pull request by montana-mil**
## Description
This PR adds functionality to the "Add new Environment" button at the bottom of the app settings table.

This also fixes a bug with the multistep modals that would freeze the site if the user canceled out of the modal.

## Pivotal Tracker
https://www.pivotaltracker.com/story/show/165516875
modal bug: https://www.pivotaltracker.com/story/show/165935002

## Screenshots
Screen Shot 2019-05-13 at 11 42 22 AM
Screen Shot 2019-05-13 at 11 42 56 AM
Screen Shot 2019-05-13 at 11 44 19 AM

### CODE REVIEW

File: atst/utils/context_processors.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 
 from atst.database import db
 from atst.domain.authz import Authorization
+from atst.domain.exceptions import NotFoundError
```
Comment (dandds) [inline atst/utils/context_processors.py]: Is this being used?

File: templates/fragments/applications/add_new_environment.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
+{% from "components/alert.html" import Alert %}
+{% from 'components/save_button.html' import SaveButton %}
+{% from "components/text_input.html" import TextInput %}
+{% from "components/toggle_list.html" import ToggleButton, ToggleSection %}
+
+<form method='POST' id="add-new-env" action='{{ url_for("applications.new_environment", application_id=application.id) }}' autocomplete="off" enctype="multipart/form-data">
```
Comment (dandds) [inline templates/fragments/applications/add_new_environment.html]: NIt-picky, but I think form tags should surround as few elements as possible. Can you try putting it _just_ inside the the `v-if="open"` div?

File: atst/utils/context_processors.py (hunk 1 above)
Comment (montana-mil) [reply atst/utils/context_processors.py]: yep! https://github.com/dod-ccpo/atst/pull/815/files#diff-5996091612960aefbff6c331f3c40a27R66

File: templates/fragments/applications/edit_team.html [hunk 3]
```diff
@@ -1,4 +1,5 @@
 {% from "components/options_input.html" import OptionsInput %}
+{% from "components/toggle_list.html" import ToggleButton, ToggleSection %}
```
[the remark this answers is not in the document]
Comment (montana-mil) [reply templates/fragments/applications/edit_team.html]: i think this comment was supposed to be on this line: https://github.com/dod-ccpo/atst/pull/815/commits/2284aa46af0638180d27be04d13761e211c2efbe#diff-f41abca667d591a25cd4acac38c3fb76R4

removing it now!

[the remark this answers is not in the document]
Comment (dandds) [reply templates/fragments/applications/edit_team.html]: whoooops

File: atst/utils/context_processors.py (hunk 1 above)
Comment (dandds) [reply atst/utils/context_processors.py]: oooh right, this was the bug you uncovered

**Pull request merged**